tests/test_base_resources.py

```diff
@@ -911,3 +911,75 @@ def test_initial_portfolio_metrics(self, reqmock):
                     text='{"num": 1.0}')
         f = OptimizationView(id='abc123').initial_metrics()
         assert f.num == 1.0
+
+    sensitivity_text = """
+    {"sensitivities": [
+    {"normalized_interquartile_range": 0.25, "min": 0.0, "normalized_standard_deviation": 0.29,
+    "initial_share": 0.18, "hist": { "0.05": 5, "0.1": 0, "0.15": 0, "0.2": 0, "0.25": 0, "0.3": 9, "0.35": 0,
+    "0.4": 0, "0.45": 0, "0.5": 3, "0.55": 0, "0.6": 5, "0.65": 0, "0.7": 3, "0.75": 0, "0.8": 3, "0.85": 2,
+    "0.9": 4, "0.95": 2, "1.0": 64}, "mean": 0.81, "max": 1.0, "ref_id": "layer2"
+    },
+    { "normalized_interquartile_range": 0.71, "min": 0.0, "normalized_standard_deviation": 0.35,
+    "initial_share": 0.08, "hist": {"0.05": 36, "0.1": 0, "0.15": 8, "0.2": 1, "0.25": 1, "0.3": 3,
+    "0.35": 2, "0.4": 3, "0.45": 0, "0.5": 0, "0.55": 8, "0.6": 0, "0.65": 12, "0.7": 0, "0.75": 1,
+    "0.8": 10, "0.85": 0, "0.9": 7, "0.95": 0, "1.0": 8}, "mean": 0.37, "max": 1.0, "ref_id": "layer1" }
+    ] } """
+
+    def test_sensitivity_analysis_results(self, reqmock):
```

Comment:
These tests seems to be very similar to each other, maybe you can combine them into a parametrization test `@pytest.mark.parametrize`.  You can find some examples in api unit tests

Reply:
Created one parameterized test instead of three similar tests. Last test is a little different, kept it separate.